wasmapi/go/map.go

```diff
@@ -88,6 +88,10 @@ const (
 	RingBuf
 	InodeStorage
 	TaskStorage
+	BloomFilter
+	UserRingbuf
+	CgroupStorage
+	Arena
 )
```

Comment:
LGTM

- [BloomFilter](https://github.com/torvalds/linux/commit/9330986c03006ab1d33d243b7cfe598a7a3c1baa) takes a NULL pointer as key, so it won't work but the kernel will just return an error and it is safe.
- [UserRingbuf](https://github.com/torvalds/linux/commit/583c1f420173f7d84413a1a1fbf5109d798b4faa) requires separate bpf helpers. The current lookup/update/delete helpers will safely return an error.
- [CgroupStorage](https://github.com/torvalds/linux/commit/de9cbbaadba5adf88a19e46df61f7054000838f6) takes the cgroup inode id as key but not its fd, so the marshalling will work fine.
- [Arena](https://github.com/torvalds/linux/commit/317460317a02a1af512697e6e964298dedd8a163)'s current lookup/update/delete helpers will safely return an error (-EOPNOTSUPP or -EINVAL).